### PR TITLE
Write Cypress test: "MM-T1960 Marketplace is available when "Enable Plugins" is true"

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

I fixed some "Marketplace should not be visible" tests, which seemed to have been altered in [this PR](https://github.com/mattermost/mattermost-webapp/commit/fc4e5323de1a9f3f4e8e3526584ebd156ffd7b29#diff-91137a34d5c4d9c156f392f16f2bd819) to not check for visibility. I also implemented the [Marketplace is available when “Enable Marketplace” is set to true](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T1958) test.

The two tests I added are identical

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-27831

### CODE REVIEW

File: e2e/cypress/integration/plugins/marketplace/marketplace_spec.js [hunk 1]
```diff
@@ -23,7 +23,20 @@ describe('Plugin Marketplace', () => {
...
+                // # Click hamburger main menu
+                cy.get('#sidebarHeaderDropdownButton').click();
+
+                // * Dropdown menu should be visible
+                cy.get('.dropdown-menu').should('be.visible').within(() => {
+                    // * Plugin Marketplace button should not be visible
+                    cy.findByText('Plugin Marketplace').should('not.be.visible');
+                });
+            });
+        });
```
[the remark this answers is not in the document]
Comment (mickmister) [reply e2e/cypress/integration/plugins/marketplace/marketplace_spec.js:37]: @marianunez Indeed they are. Thanks for pointing this out :)

File: e2e/cypress/integration/plugins/marketplace/marketplace_spec.js [hunk 2]
```diff
@@ -86,6 +87,53 @@ describe('Plugin Marketplace', () => {
...
+                PluginSettings: {
+                    Enable: true,
+                    EnableMarketplace: true,
+                    MarketplaceUrl: 'https://api.integrations.mattermost.com',
+                },
+            });
+            cy.visit(townsquareLink);
+        });
+
+        it('MM-T1960 Marketplace is available when "Enable Marketplace" is set to true', () => {
```
Comment (prapti) [inline e2e/cypress/integration/plugins/marketplace/marketplace_spec.js:120]: Also could this be changed to MM-T1958 to match the test case ID?
```suggestion
        it('MM-T1958 Marketplace is available when "Enable Marketplace" is set to true', () => {
```

**Review by marianunez — APPROVED** [review]
LGTM. Thanks @mickmister!

**Review by prapti — APPROVED** [review]
LGTM! Thank you so much @mickmister!

**Review by DHaussermann — APPROVED** [review]
Already reviewed by @prapti

**Pull request merged**